### ISSUE TITLE
brew: yq@3 has been disabled because it is not maintained upstream

### ISSUE THREAD

**Issue by bitsofinfo** [mikefarah/yq#995]
```
brew install yq@3
Error: yq@3 has been disabled because it is not maintained upstream!
```

**Comment by mikefarah** [comment]
Yep that's right, as of August this year I am no longer maintaining v3 of `yq` - I highly recommend updating to v4 as v3 hasn't been touched since I released it (which means it hasn't had any dependency updates nor rebuilds with a newer go compiler to fix potential security vulnerabilities. Check out the guide here https://mikefarah.gitbook.io/yq/upgrading-from-v3 - it should be pretty straight forward to update.
